go.mod

```diff
@@ -34,6 +28,8 @@ require (
 )
 
 replace (
+	git.apache.org/thrift.git => github.com/apache/thrift v0.12.0
```

Comment:
this is already part of the master https://github.com/codeready-toolchain/host-operator/blob/master/go.mod#L37

Reply:
I've merged master